### PR TITLE
standalone false

### PR DESCRIPTION
**Pull request by Plus17**
related issue https://github.com/joshnuss/xml_builder/issues/49

## Changes
- `standalone` option default vaule now is `nil` instead of `false`
- Pass a `false` value to `standalone` option now generate `standalone="no"`
- Update the readme with this behavior change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,6 +184,15 @@ Outputs.
 <outsider/>
 ```
 
+If otherwise you need `standalone ="false"` in the XML declaration, you can pass `standalone: false` as an option to the` generate / 2` call.
+
+Outputs.
+
+```xml
+<?xml version="1.0" standalone="false"?>
```
Comment (joshnuss) [inline README.md:192]: Did you mean `standalone="no"`? I believe valid values are "yes" or "no"

File: test/xml_builder_test.exs [hunk 2]
```diff
@@ -109,6 +109,11 @@ defmodule XmlBuilderTest do
       assert XmlBuilder.generate(:xml_decl, standalone: true) == expectation
     end
 
+    test "standalone option is used with false value" do
+      expectation = ~s|<?xml version="1.0" encoding="UTF-8" standalone="no"?>|
+      assert XmlBuilder.generate(:xml_decl, standalone: false) == expectation
+    end
```
Comment (joshnuss) [inline test/xml_builder_test.exs:115]: Should we add a test for `standalone: nil`?

**Review by joshnuss — CHANGES_REQUESTED** [review]
Thanks @Plus17, it looks good!

Left 2 small comments

File: README.md (hunk 1 above)
Comment (Plus17) [reply README.md]: My bad 😅

File: test/xml_builder_test.exs (hunk 2 above)
Comment (Plus17) [reply test/xml_builder_test.exs:115]: Sure! 👍

**Review by joshnuss — APPROVED** [review]

**Comment by joshnuss** [comment]
Thanks @Plus17!
